openmls/src/group/core_group/test_proposals.rs

```diff
@@ -353,7 +357,9 @@ fn test_group_context_extensions(ciphersuite: Ciphersuite, backend: &impl OpenMl
         ProposalType::Remove,
         ProposalType::Update,
     ];
-    let required_capabilities = RequiredCapabilitiesExtension::new(extensions, proposals);
+    let credentials = &[CredentialType::Basic];
```

Comment:
Generally this shouldn't be necessary because "basic" is a "default" credential and should therefore always be supported. But let's file a follow up to handle the "default" credentials.

Reply:
https://github.com/openmls/openmls/issues/1204